main.go

```diff
@@ -51,21 +54,40 @@ func main() {
 		g.Procs[pc.Name] = &process{Config: pc}
 	}
 
-	// Listen for a kill signal. When one comes, kill all running children
-	// before exiting.
+	// Listen for signals and respond appropriately.
 	var sigs = make(chan os.Signal)
-	signal.Notify(sigs, unix.SIGTERM, unix.SIGINT)
+	signal.Notify(sigs, unix.SIGTERM, unix.SIGINT, unix.SIGINFO)
 	go func() {
-		var sig = <-sigs
-		log.Println("Signal", sig, " received, termiating children...")
-		for i := range g.Procs {
-			if g.Procs[i].Running {
-				log.Println("killing", g.Procs[i].Config.Name)
-				unix.Kill(g.Procs[i].Status.Pid, unix.SIGTERM)
+		for true {
+			var sig = <-sigs
+			switch sig {
+			// In the case of a kill signal, terminate all
+			// children before exiting.
+			case unix.SIGTERM, unix.SIGINT:
+				log.Println("Signal", sig, " received, termiating children...")
+				for i := range g.Procs {
+					if g.Procs[i].Running {
+						log.Println("killing", g.Procs[i].Config.Name)
+						unix.Kill(g.Procs[i].Status.Pid, unix.SIGTERM)
+					}
+				}
+				log.Println("done")
+				os.Exit(0)
+			case unix.SIGINFO:
+				fmt.Println("The following children are running:")
+				fmt.Fprintln(writer, "Process\tPath\tArgs\tDir\tUser\tGroup")
+				fmt.Fprintln(writer, "-------\t----\t----\t---\t----\t-----")
+				for i := range g.Procs {
+					if !g.Procs[i].Running {
```

Comment:
All proccesses in the config file should have a line in this output, including failed or completed processes. For now, just add a column for running.